app/grandchallenge/components/tasks.py

```diff
@@ -120,6 +121,8 @@ def validate_docker_image(*, pk: uuid.UUID, app_label: str, model_name: str):
                 instance=instance
             )
             instance.is_manifest_valid = True
+            if mark_as_desired:
+                instance.mark_desired_version()
```

Comment:
On second thought. maybe this should be moved to the end of this task, when the image is also in the registry and on sagemaker. Otherwise, we end up with a short period of downtime when uploading a new image where there is no active image after manifest validation and upload to the registry. What do you think?